src/pages/signatures/guides/batch-signatory.mdx

```diff
@@ -0,0 +1,46 @@
+---
+product: signatures
+category: Guides
+sort: 20
+title: Batch Signatory
+---
+
+import GraphQLExample from '../../../components/GraphQLExample';
+
+**Batch Signatory** is a concept for bundling multiple signatories.
+Previously each signatory would have to operated individually, but now one can perform batch operations in a single call and thereby avoiding repeated authentications or rejections.
+
+A batch signatory can be created by providing a list of pairs of signature order and signatory IDs.
+
+It is _important_ to note that each signature order and signatory **require** sharing at least one evidence provider in order to work.
```

Comment:
Suggestion:
Each signature order and signatory **must share** at least one evidence provider.